source/renderer/app/components/wallet/settings/WalletRecoveryPhraseVerificationWidget.js

```diff
@@ -235,16 +254,19 @@ export default class WalletRecoveryPhraseVerificationWidget extends Component<Pr
       <div className={styles.component}>
         <h2>{intl.formatMessage(messages.title)}</h2>
         <div className={styles.description}>
-          {intl.formatMessage(messages.description, {
-            wordCount,
-          })}{' '}
-          <Tooltip
-            className={styles.paperWallet}
-            skin={TooltipSkin}
-            tip={intl.formatMessage(messages.paperWalletDescription)}
-          >
-            {intl.formatMessage(messages.paperWalletTitle)}
-          </Tooltip>
+          {intl.formatMessage(messages.description, { wordCount })}
+          {!isIncentivizedTestnet && (
```

Comment:
@daniloprates it is possible to restore a paper wallet on the ITN too. Therefore, I believe we don't need this switch...

Reply:
hey @nikolaglumac, for ITN we only display this widget for Shelley wallets, that's why we need the conditional

Reply:
oh I forgot that 🤦 sorry!